app/controllers/katello/api/v2/systems_controller.rb

```diff
@@ -388,8 +388,13 @@ def setup_content_view(cv_id)
     end
 
     def systems_by_errata(errata_uuids, installable, non_installable)
-      installable = installable.nil? ? false : installable.to_bool
-      non_installable = non_installable.nil? ? false : non_installable.to_bool
+      if !!installable != installable
+        installable = installable.nil? ? false : installable.to_bool
+      end
+
+      if !!non_installable != non_installable
+        non_installable = non_installable.nil? ? false : non_installable.to_bool
+      end
```

Comment:
Hah, well I could just delete them but I didn't want to chance breaking anything for upcoming releases.  Thoughts?  @jlsherrill